kube/Cargo.toml

```diff
@@ -64,6 +64,10 @@ async-compression = { version = "0.3.7", features = ["gzip", "tokio"], optional
 hyper-timeout = "0.4.1"
 tame-oauth = { version = "0.4.7", features = ["gcp"], optional = true }
 pin-project = "1.0.4"
+rand = { version = "0.8.3", optional = true }
+# TODO Remove this and use `tungstenite`'s `derive_accept_key` when 0.13.0 is released.
+# https://github.com/snapview/tungstenite-rs/pull/174
+sha-1 = { version = "0.9", optional = true }
```

Comment:
`tungstenite` depends on `rand` and `sha-1`, so these are not new dependencies. We can also remove `sha-1` when the new version of `tungstenite` is released with the function we need.